src/Kerbero.WebApi/Controllers/GetNukiSmartLockController.cs

```diff
@@ -0,0 +1,33 @@
+using Kerbero.Domain.Common.Interfaces;
+using Kerbero.Domain.NukiActions.Interactors;
+using Kerbero.Domain.NukiActions.Models;
+using Kerbero.WebApi.Utils.Extensions;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Kerbero.WebApi.Controllers;
+
+[ApiController]
+[Route("nuki/smartlock")]
```

Comment:
```suggestion
[Route("api/nuki/smartlock")]
```
perchè non c'è classico prefisso api?